RoaringBitmap/src/main/java/org/roaringbitmap/buffer/MappeableArrayContainer.java

```diff
@@ -449,6 +449,8 @@ public boolean equals(Object o) {
       return true;
     } else if (o instanceof MappeableRunContainer) {
       return o.equals(this);
+    } else if (o instanceof MappeableBitmapContainer) {
```

Comment:
This cannot happen. An ArrayContainer can never be equal to a BitmapContainer when the data abides by the specification.